torch/_inductor/cudagraph_trees.py

```diff
@@ -1299,9 +1326,10 @@ def debug_assert_invariants(
                     self.path_weakrefs[depth][output_idx]
                 )
 
-                if is_live(self.path_weakrefs[depth][output_idx]):
-                    stor_data_ptr = self.path_weakrefs[depth][output_idx].data_ptr()
-                    stor_weak_ptr = self.path_weakrefs[depth][output_idx]()
+                w = self.path_weakrefs[depth][output_idx]
+                if w is not None and is_live(w):
```

Comment:
we could change this line and https://github.com/pytorch/pytorch/pull/105151/files#diff-3e1199aa4b118c07d8c38c05f1ced6d2c87b6b94f868fbc7596d27c06d5a8f9bR1257

If we updated `is_live` to take in `Optional[StorageWeakRefWrapper`]

Reply:
I can't do this as is, but I think the walrus operator will come in handy here.